alts/src/main/proto/grpc/gcp/handshaker.proto

```diff
@@ -229,6 +229,6 @@ service HandshakerService {
   // service expects to respond. Client does not have to wait for service's
   // response before sending next request.
   rpc DoHandshake(stream HandshakerReq)
-    returns (stream HandshakerResp) {
+      returns (stream HandshakerResp) {
```

Comment:
This diff is superfluous - due to whitespace changes. Can we get rid of it?

Reply:
This is a pure copy of the proto in https://github.com/grpc/grpc-proto. If we revert this whitespace change it will cause a pure-whitespace change in other repos.